encoders/nlp/VSETextEncoder/requirements.txt

```diff
@@ -1,4 +1,4 @@
 jina
```

Comment:
This is because it's using `FROM pytorch/pytorch:latest`, right?

Reply:
yes